meta-mbl-distro/recipes-core/packagegroups/packagegroup-mbl-production.bb

```diff
@@ -56,6 +56,7 @@ PACKAGEGROUP_MBL_PRODUCTION_PKGS_append = " connman"
 PACKAGEGROUP_MBL_PRODUCTION_PKGS_append = " connman-client"
 PACKAGEGROUP_MBL_PRODUCTION_PKGS_append = " systemd-net-conf"
 PACKAGEGROUP_MBL_PRODUCTION_PKGS_append = " systemd-watchdog-conf"
+PACKAGEGROUP_MBL_PRODUCTION_PKGS_append = " swupdate"
```

Comment:
I think this is okay for now, but when we integrate swupdate with our firmware update solution, we should remove this and just make swupdate an rdependency of our `mbl-cloud-client` recipe (or our `updated` recipe once we have one).